32_gameoflife.py

```diff
@@ -0,0 +1,87 @@
+# S30 Big N Problem #32 {Medium}
+
+# Game of Life
+
+# Time Complexity : O(n*m) n= no. of rows, m= no. of columns
+# Space Complexity : O(1) 
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : No 
+
+# Approach 
+# Traverse through the each element in matrix
+# Based on the rules, if a live cell becomes dead mark it with 2
+# if a dead cell becomes live mark it with 4
+# replace the 2 and 4 with appropriate values 
+
+
+class Solution:
+    def gameOfLife(self, board: List[List[int]]) -> None:
+        """
+        Do not return anything, modify board in-place instead.
+        """
+        
+        m=len(board)
+        n=len(board[0])
+        
+        
+        for i in range(0,m,1):
+            for j in range(0,n,1):
+                
+                live=0
+                
+                #left
+                if j-1>=0 and (board[i][j-1]==1 or board[i][j-1]==2):
```

Comment:
Please use a directions array, makes the code easily readable.

Reply:
yes, will resolve it using the directions array.